src/ldp/http/SimpleBodyParser.ts

```diff
@@ -0,0 +1,52 @@
+import { BodyParser } from './BodyParser';
+import { HttpRequest } from '../../server/HttpRequest';
+import { Quad } from 'rdf-js';
+import { QuadRepresentation } from '../representation/QuadRepresentation';
+import { RepresentationMetadata } from '../representation/RepresentationMetadata';
+import { StreamParser } from 'n3';
+import { TypedReadable } from '../../util/TypedReadable';
+import { UnsupportedMediaTypeHttpError } from '../../util/errors/UnsupportedMediaTypeHttpError';
+import 'jest-rdf';
+
+export class SimpleBodyParser extends BodyParser {
+  private static readonly contentTypes = [
+    'application/n-quads',
+    'application/trig',
+    'application/n-triples',
+    'text/turtle',
+    'text/n3',
+  ];
+
+  public async canHandle(input: HttpRequest): Promise<void> {
+    const contentType = input.headers['content-type'];
+
+    if (contentType && !SimpleBodyParser.contentTypes.some((type): boolean => contentType.includes(type))) {
```

Comment:
Is this possible?:
```suggestion
    if (contentType && !SimpleBodyParser.contentTypes.some(contentType.includes) {
```

Reply:
Doesn't work. I thought that it would have to be `contentType.includes.bind(contentType)` but there a few tests also fail so not immediately sure what it would have to be